GPyOpt/util/arguments_manager.py

```diff
@@ -103,38 +103,39 @@ def model_creator(self,model_type, exact_feval, space):
         # --------
         # --- Initialize GP model with MCMC on the parameters
         # --------
-        elif self.model_type == 'GP_MCMC':
-            self.n_samples = self.kwargs.get('n_samples',10)
-            self.n_burnin = self.kwargs.get('n_burnin',100)
-            self.subsample_interval = self.kwargs.get('subsample_interval',10)
-            self.step_size = self.kwargs.get('step_size',1e-1)
-            self.leapfrog_steps = self.kwargs.get('leapfrog_steps',20)
-            return  GPModel_MCMC(self.kernel, self.noise_var, self.exact_feval, self.n_samples, self.n_burnin, self.subsample_interval, self.step_size, self.leapfrog_steps, self.verbosity_model)
+        elif model_type == 'GP_MCMC':
+            n_samples = self.kwargs.get('n_samples',10)
+            n_burnin = self.kwargs.get('n_burnin',100)
+            subsample_interval = self.kwargs.get('subsample_interval',10)
+            step_size = self.kwargs.get('step_size',1e-1)
+            leapfrog_steps = self.kwargs.get('leapfrog_steps',20)
+            return  GPModel_MCMC(kernel, noise_var, exact_feval, n_samples, n_burnin, subsample_interval, step_size, leapfrog_steps, verbosity_model)
 
         # --------
         # --- Initialize RF: values taken from default in scikit-learn
         # --------
-        elif self.model_type =='RF':
-            if 'verbosity_model' in self.kwargs: self.verbosity = self.kwargs['verbosity']
-            else: self.verbosity = False
-            return RFModel(verbose=self.verbosity_model)
+        elif model_type =='RF':
+            #if 'verbosity_model' in self.kwargs: self.verbosity = self.kwargs['verbosity']
```

Comment:
Why are these lines commented out now?

Reply:
Hi! The variable 'verbosity_model' is assigned in line 88 and then used in line 120. I commented lines 118 and 119 because they seem redundant. I might have mistaken, but it looks like 'self.verbosity' is not needed here.

Reply:
That's actually right. Can you please just remove these lines instead of commenting them out?